da.go

```diff
@@ -14,10 +14,10 @@ type DA interface {
 	Get(ctx context.Context, ids []ID) ([]Blob, error)
 
 	// GetIDs returns IDs of all Blobs located in DA at given height.
-	GetIDs(ctx context.Context, height uint64) ([]ID, error)
+	GetIDs(ctx context.Context, height uint64, namespace Namespace) ([]ID, error)
 
 	// Commit creates a Commitment for each given Blob.
-	Commit(ctx context.Context, blobs []Blob) ([]Commitment, error)
+	Commit(ctx context.Context, blobs []Blob, namespace Namespace) ([]Commitment, error)
```

Comment:
minor: given that namespace needs to be passed to the above functions as the last param, should we modify Submit for consistency, e.g.:

```Submit(ctx context.Context, blobs []Blob, opts *SubmitOptions)``` to

```Submit(ctx context.Context, blobs []Blob, gasPrice float64, namespace Namespace)```

Reply:
I would like this better. Option structs are generally bad devex